src/messages/responses/cache-dictionary-fetch.ts

```diff
@@ -14,47 +14,58 @@ export abstract class Response extends ResponseBase {}
 
 class _Hit extends Response {
   private readonly items: cache_client._DictionaryFieldValuePair[];
-  private readonly dictionaryUint8ArrayUint8Array: Map<Uint8Array, Uint8Array> =
-    new Map();
-  private readonly dictionaryStringString: Map<string, string> = new Map();
-  private readonly dictionaryStringArrayBuffer: Map<string, Uint8Array> =
-    new Map();
   private readonly _displayListSizeLimit = 5;
 
   constructor(items: cache_client._DictionaryFieldValuePair[]) {
     super();
     this.items = items;
   }
 
-  public valueDictionaryUint8ArrayUint8Array(): Map<Uint8Array, Uint8Array> {
-    for (const item of this.items) {
-      this.dictionaryUint8ArrayUint8Array.set(item.field, item.value);
-    }
-    return this.dictionaryUint8ArrayUint8Array;
+  public valueMap(): Map<string, string> {
+    return this.valueMapStringString();
   }
```

Comment:
Curious if anyone has any objections to this alias, which hopefully would be the most common use case?

Reply:
Will non-text fields be decoded into garbage strings? I'm alright with it as long as there won't be surprise errors thrown.

Reply:
`string | Uint8Array` I think would be harmful - there's nothing in the backing data that would hint to the sdk which is appropriate, so it would vary per field 😬.

Reply:
Fair. If you try to mix byte array and string fields in a dictionary then you can't expect intuitive things to happen.